js/renovation/ui/responsive_box/_toDelete/responsive_box.js

```diff
@@ -112,18 +111,6 @@ const ResponsiveBox = CollectionWidget.inherit({
 
     _initMarkup: function() {
         this.callBase();
-        // NOTE: Fallback box strategy
-        this._updateRootBox();
-    },
-
-    _updateRootBox: function() {
-        clearTimeout(this._updateTimer);
-
-        this._updateTimer = setTimeout((function() {
```

Comment:
Why we don't need timeout anymore?

Reply:
This timer is necessary only for Fallback strategy, which will be removed soon.
I remove timer in this request, because it uses the root property, which I removed in this PR